de.weingardt.mylyn.gitlab.core/about.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+<title>About</title>
+</head>
+<body lang="EN-US">
+<h2>About This Content</h2>
+ 
+<p>July, 2016</p>	
+<h3>License</h3>
+
+<p> Unless otherwise indicated below, all content of this plugin &quot;Content&quot; is provided to you under the terms

```

Comment:
@pweingardt usually if you look into the about.html of eclipse plugins. The author is emntiones as well in a way such as "The content of this plugin is made available by pweingart". Was not sure if you would like it or not. Please check if you like it or not.